fern/apis/server/definition/ingestion.yml

```diff
@@ -26,23 +26,71 @@ types:
       observation-create: ObservationCreateEvent
       observation-update: ObservationUpdateEvent
 
+  ObservationEvent:
```

Comment:
ObservationEvent -> Observation / ObservationBody

Reply:
Does not fit to the zod validation on this event type, include the different observation types in the openapi spec as well?